conan/tools/gnu/autotoolstoolchain.py

```diff
@@ -153,6 +153,7 @@ def environment(self):
         env.append("CXXFLAGS", self._filter_list_empty_fields(self.cxxflags))
         env.append("CFLAGS", self._filter_list_empty_fields(self.cflags))
         env.append("LDFLAGS", self._filter_list_empty_fields(self.ldflags))
+        env.define("PKG_CONFIG_PATH", self._conanfile.generators_folder)
```

Comment:
I am not sure we want the ``autotoolstoolchain`` to completely re-define the PKG_CONFIG_PATH. In case there are some other .pc files in some other place and the user has PKG_CONFIG_PATH defined, we want to respect that. So it seems ``generators_folder`` should probably be pre-pended to make the Conan generated files a priority, but not remove the existing value.